boltdesignsystem/bolt
# DS-579 Header Height HTML Attribute

## Jira

https://pegadigitalit.atlassian.net/browse/DS-579

## Summary

A `data-header-height` attr was added to the `<header />` 

## Details

- The height of the `bolt-page-header` component is added to the `data-header-height` attr
- The `data-header-height` attr updates on resizing when the height of the header changes

## How to test

Pull the branch. Check if `data-header-height` attr is added properly to the `bolt-page-header` component. Check if this attr updates when the component's height changes on window resize.

## Release notes

`data-header-height` attribute is added to the `bolt-page-header` componenet.
